app/components/shared/card_component.rb

```diff
@@ -0,0 +1,41 @@
+class Shared::CardComponent < GovukComponent::Base
+  include ViewComponent::SlotableV2
+
+  renders_many :action_items, "ActionItemComponent"
```

Comment:
I'd rather `renders_many :actions` with lamba components (and similar for body and header)